src/dom/JsValueConversion.ts

```diff
@@ -5,6 +5,13 @@ import { IonTypes } from "../IonTypes";
 import { _hasValue } from "../util";
 import { Value } from "./Value";
 
+// In ./Boolean.ts and ./String.ts, the native Boolean and String types
+// are shadowed by the export class defined in the file, but said class
+// definitions still need to reference the original class definitions.
+// This provides a mechanism for doing so.
+export const _NativeJsBoolean = Boolean;
+export const _NativeJsString = String;
+
```

Comment:
I initially tried tackling this with

(String.ts)
```
export class String extends Value(
  global.String,
  IonTypes.STRING,
  _fromJsConstructor
) {
 ...
}
```
->
```
class StringImpl extends Value(
  String,
  IonTypes.STRING,
  _fromJsConstructor
) {
  ...
}
export { StringImpl as String };
```

But swapping the directly exported variable from the class itself to a constant screwed up the generated documentation.